templates/profile.html

```diff
@@ -203,6 +216,42 @@ <h4 class="modal-title" id="myModalLabel">Disconnect Account: {{provider_full_na
     </div>
 </div>
 {% endfor %}
+<div class="modal fade" id="change-password" tabindex="-1" role="dialog">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+              <h4 class="modal-title" id="myModalLabel">Change Password</h4>
+            </div>
+            <form action="#" id="password-form">
+                <div class="modal-body">
+                    <p>You can change your password here.</p>
+                    <div class="form-group">
+                        <label for="old-password">Current password</label>
+                        <input type="password" id="old-password" placeholder="Enter old password" class="form-control"
+                               name="old-password">
+                    </div>
+                    <div class="form-group">
+                        <label for="old-password">New password</label>
```

Comment:
```suggestion
                        <label for="new-password1">New password</label>
```

Should match the field.

Reply:
What about `new-password`? And `new-password-confirmation` for the confirmation?

Reply:
Either way is fine, it just shouldn't be `old-password` for all three labels.

Reply:
Hmm, thought I changed it after the copy-paste. Apparently not. Thanks!
I think I went with `new-password1` and `new-password2` after the registration dialog, but I'll recheck.

Reply:
Changed the names/ids of the password inputs and fixed the label assignment.